docs/technical_reference/unit_models/index.rst

```diff
@@ -19,5 +19,6 @@ Unit Models
    gac
    osmotically_assisted_reverse_osmosis_0D
    anaerobic_digestor
+   translator_adm1_asm1
```

Comment:
I think this should be under property models instead of unit models. See comment below.